config/locales/en.txt

```diff
@@ -1507,6 +1507,7 @@
   form_names_refs_help: Each line is considered a separate reference, but no other formatting is enforced.
   name_error_unregistrable: "The [rank] '[name]' is not eligible for an ICN identifier."
   name_error_icn_id_in_use: "An ICN identifier number may be used by only one Name. '[number]' is already used by '[name]'."
+  name_error_icn_id_non_numeric: "ICN ID must be numeric"
```

Comment:
Minor I know, but I feel like "must be numeric" is programmer-talk.  Maybe better to spell it out in English: "ICN ID must be a number".

Reply:
Thanks! 
I think I ended up not using this string. Will double check, and either delete it or turn it to English.